docs/TUTORIAL.md

```diff
@@ -433,34 +424,41 @@ new metadata to disk.
 ```python
 # Continuing from the previous section . . .
 
-# Remove a target file listed in the "targets" metadata.  The target file is
-# not actually deleted from the file system.
->>> repository.targets.remove_target("repository/targets/file3.txt")
```

Comment:
is this line still needed? As currently written I don't think any files are removed

Reply:
Thanks for spotting this, @mnm678! I accidentally removed this in a recent commit. It's back now.